sqldb/sqlc/migrations/000005_migration_tracker.up.sql

```diff
@@ -0,0 +1,17 @@
+-- migration_tracker is a table that keeps track of custom migrations that have
+-- been run on the database. This table is used to ensure that migrations are
+-- idempotent and are only run once.
+CREATE TABLE IF NOT EXISTS migration_tracker (
+        -- migration_id is the id of the migration.
+        migration_id TEXT PRIMARY KEY,
```

Comment:
still getting to that part of the PR but assuming that the order of migrations is kept track of at a code level then if this is text based?

Reply:
As we discussed offline, I've transitioned to a new model that uses a global database version represented as an integer. The complete list of migrations, including future in-code ones, is now defined in sqldb/migrations.go. This table will track which migrations have been successfully executed.